grafana/Dockerfile

```diff
@@ -16,7 +16,9 @@ USER root
 RUN grafana-cli --pluginsDir "${GF_PATHS_PLUGINS}" plugins install pr0ps-trackmap-panel 2.1.2 && \
     grafana-cli --pluginsDir "${GF_PATHS_PLUGINS}" plugins install natel-discrete-panel 0.1.0 && \
     grafana-cli --pluginsDir "${GF_PATHS_PLUGINS}" plugins install grafana-piechart-panel 1.6.1 && \
-    grafana-cli --pluginsDir "${GF_PATHS_PLUGINS}" --pluginUrl https://github.com/panodata/grafana-map-panel/releases/download/0.15.0/grafana-map-panel-0.15.0.zip plugins install grafana-worldmap-panel-ng
+    grafana-cli --pluginsDir "${GF_PATHS_PLUGINS}" --pluginUrl https://github.com/panodata/grafana-map-panel/releases/download/0.15.0/grafana-map-panel-0.15.0.zip plugins install grafana-worldmap-panel-ng && \
+    chown -R "grafana" "${GF_PATHS_PLUGINS}" && \
+    chmod -R 777 "${GF_PATHS_PLUGINS}"
```

Comment:
The [custom Dockerfile ](https://github.com/grafana/grafana/blob/main/packaging/docker/custom/Dockerfile) suggests to change the ownership of the pluginsDir even before installing the plugins:

```bash
USER root

RUN mkdir -p "$GF_PATHS_PLUGINS" && \
    chown -R grafana "$GF_PATHS_PLUGINS"

USER grafana

RUN grafana-cli --pluginsDir "${GF_PATHS_PLUGINS}" plugins install pr0ps-trackmap-panel 2.1.2 && \
    grafana-cli --pluginsDir "${GF_PATHS_PLUGINS}" plugins install natel-discrete-panel 0.1.0 && \
    grafana-cli --pluginsDir "${GF_PATHS_PLUGINS}" plugins install grafana-piechart-panel 1.6.1 && \
    grafana-cli --pluginsDir "${GF_PATHS_PLUGINS}" --pluginUrl https://github.com/panodata/grafana-map-panel/releases/download/0.15.0/grafana-map-panel-0.15.0.zip plugins install grafana-worldmap-panel-ng

COPY logo.svg /usr/share/grafana/public/img/grafana_icon.svg
# ...
```

It also seems unnecessary to change the permissions that way. What do you think? Would that work for you as well?

Reply:
Both should produce identical results regarding the owner of the folder and files, so yes it will work for me.

The only difference will be the missing chmod 777, but I don't think it's needed in that case. I only suggested it because it is included in the "default" Dockerfile [default Dockerfile](https://github.com/grafana/grafana/blob/main/Dockerfile) line 78:
`    chmod -R 777 "$GF_PATHS_DATA" "$GF_PATHS_HOME/.aws" "$GF_PATHS_LOGS" "$GF_PATHS_PLUGINS" "$GF_PATHS_PROVISIONING"`

Thanks !

Reply:
All right, I see. Let's try this approach for now then 👍